models/comment.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection.js");
+
+exports.delComment = async (id) => {
+  const { rows: allComs } = await db.query("SELECT * FROM comments;");
+
+  if (id > allComs.length) {
```

Comment:
This is a very risky way of checking if an ID exists in the database, because it's assuming that the ID is always going to be according to the length. Once you start deleting comments this will fail in production. You might as well request the specific ID in the query and if it returns nothing then it means it doesn't exist.